src/pages/Home/Home.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../styles/variables.scss';
+@import '../../styles/mixins.scss';
+
+.home__page {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+
+    &__logo {
+        margin-bottom: 4rem;
+    }
+}
+
+@include for-phone-only {
+    .home__page {
+        &__logo {
```

Comment:
shrink logo more, would be nicer I think

Reply:
width = 60% be better for u?